addons/l10n_se_ocr/views/partner_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="res_partner_ocr_form" model="ir.ui.view">
+            <field name="name">res.partner.ocr.form</field>
+            <field name="model">res.partner</field>
+            <field name="inherit_id" ref="base.view_partner_form"/>
+            <field name="arch" type="xml">
+                <group name="accounting_entries" position="after">
+                    <group string="Payment Options" name="payment_options">
```

Comment:
As this is a new group you could add Sweden somewhere in it, so it is clear what it is about.

Reply:
Well to be honest it could be used in Norway as well. The function is the same however the name is different. It's called KID in Norway.

Reply:
We can check later for making it generic.  For now, it is good to know for the user it is related to the Sweden localization.